editor/test/actions.js

```diff
@@ -1,70 +1,474 @@
 /**
  * Internal dependencies
  */
-import {
-	focusBlock,
-	replaceBlocks,
-	startTyping,
-	stopTyping,
-	requestMetaBoxUpdates,
-	handleMetaBoxReload,
-	metaBoxStateChanged,
-	initializeMetaBoxState,
-	fetchReusableBlocks,
-	updateReusableBlock,
-	saveReusableBlock,
-	convertBlockToStatic,
-	convertBlockToReusable,
-} from '../actions';
+import * as actions from '../actions';
```

Comment:
I don't think using wildcard makes it easier as you need to prefix all method with `actions.`. It's also a common pattern used to import everything explicitly. It isn't as important in the tests context but can make a difference when bundling code for the browser.